client/src/main/java/com/paypal/selion/configuration/ConfigManager.java

```diff
@@ -54,35 +54,31 @@
 public final class ConfigManager {
 
     // Map for each TestNG test name -> local SeLion config associated with <test>
-    private static Map<String, LocalConfig> configsMap = new ConcurrentHashMap<String, LocalConfig>();
+    private static Map<Thread, LocalConfig> configsMap = new ConcurrentHashMap<Thread, LocalConfig>();
 
     private ConfigManager() {
         // Utility class. So hide the constructor
     }
 
     /**
-     * Adds the local configuration {@link LocalConfig} associated with name. Over-rides any config with the same name.
+     * Adds the local configuration {@link LocalConfig} associated with thread. Over-rides any config with the same thread.
      * 
      * @param config
      *            The LocalConfig.
-     * @param name
-     *            The name to associate with local config.
      */
-    public static synchronized void addConfig(String name, LocalConfig config) {
-        SeLionLogger.getLogger().entering(new Object[] { name, config });
-        checkArgument(StringUtils.isNotBlank(name),
-                "A testname for which configuration is being added cannot be null (or) empty.");
+    public static synchronized void addConfig(LocalConfig config) {
```

Comment:
@renelux: This change will break clients.  
My recollection is test names were unique in suite files. Has that changed ?
 
Also the changes here will not work in a parallel suite like this one:
```
<suite thread-count="2" name="SeLion Tests Suite" parallel="classes">
    <parameter name="browser" value="*firefox" />
    <test name="Parallel1">
        <parameter name="browser" value="*chrome" />
        <classes>
            <class name="com.paypal.selion.configuration.A" />
        </classes>
    </test>
</suite> 
```
The class A will get firefox and not chrome browsers.